woocommerce/woocommerce-android
# Issue/3725 connected reader state

Parent issue #3725

Contains basic infra and UI for the connected state of the connection flow

Design - vaw3DvewbUwuQnPXpdNGGH-fi-16%3A673525

### How to test
Settings -> Manage reader -> Redirect -> Connect and check the UI

![image](https://user-images.githubusercontent.com/4923871/118837432-7d01ab80-b8cd-11eb-8a38-f2d195f163c1.png)

https://user-images.githubusercontent.com/4923871/118837539-986cb680-b8cd-11eb-8808-035fa9939c0b.mp4
